app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard2Activity.java

```diff
@@ -0,0 +1,237 @@
+package corp.kairos.adamastor.Onboarding;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.Criteria;
+import android.location.Location;
+import android.location.LocationManager;
+import android.location.LocationProvider;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.android.gms.maps.model.Polygon;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Set;
+
+import corp.kairos.adamastor.R;
+import corp.kairos.adamastor.Settings.Settings;
+
+import static android.location.Criteria.ACCURACY_HIGH;
+
+/**
+ * Created by jlsilva94 on 10/01/18.
```

Comment:
Please remove all creation comments such as this one.

Reply:
i was totally forgetting this, thanks for the heads up 👍